pkg/reconciler/taskrun/resources/taskresourceresolution.go

```diff
@@ -70,3 +71,25 @@ func ResolveTaskResources(ts *v1alpha1.TaskSpec, taskName string, kind v1alpha1.
 	}
 	return &rtr, nil
 }
+
+// getResource will return an instance of a PipelineResource to use for r, either by getting it with getter or by
+// instantiating it from the embedded spec.
+func getResource(r *v1alpha1.TaskResourceBinding, getter GetResource) (*v1alpha1.PipelineResource, error) {
+	// Check both resource ref or resource Spec are not present. Taskrun webhook should catch this in validation error.
```

Comment:
Very minor nit (and I see that this func has just been moved from another place, so it's totally not required to fix this now) but the block of code below this is pretty self-explanatory. This comment probably isn't needed?